src/org/whstsa/library/commands/functional/DayGeneratorCommand.java

```diff
@@ -0,0 +1,52 @@
+package org.whstsa.library.commands.functional;
+
+import org.json.JSONObject;
+import org.whstsa.library.LibraryDB;
+import org.whstsa.library.World;
+import org.whstsa.library.commands.api.ICommand;
+import org.whstsa.library.commands.api.ICommandSender;
+import org.whstsa.library.db.Loader;
+import org.whstsa.library.util.DayGenerator;
+import org.whstsa.library.util.Logger;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.List;
+
+/**
+ * Created by Andre on 12/7/17
+ */
+public class DayGeneratorCommand implements ICommand {
+
+    private static Logger LOGGER = new Logger("DayGenerator");
+
+    @Override
+    public JSONObject handle(String[] args, ICommandSender commandSender) {
+        int totalDaysAdvanced = Integer.parseInt(args[0]);
+        for (int currentDate = 0; currentDate< totalDaysAdvanced; currentDate++) {
+            LOGGER.debug("" + World.getDate());
+            LOGGER.debug();
```

Comment:
Why are you calling the `debug` method with no arguments?

Reply:
(In debug mode, i wanted to space things out especially for day generator, so I thought to use an empty Logger.debug(); )